pkg/crc/output/types.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright (C) 2019 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package output
+
+type VersionOutputStruct struct {
```

Comment:
I'm not totally convinced this to be part of output, as at the moment the pkg itself is not reponsible for filling the struct.

I'll have to 'sleep' about this...

Reply:
Right. I will also give it another thought. We might be able to find a better place for it.